### PR TITLE
Code id gov + misc fixes

### PR DESCRIPTION
**Pull request by achamayou**
All the remaining bits of JS Governance as far as I can tell, including enabling it by default for all the tests, the sandbox etc.

- add_code_id
- remove_code_id
- set_service_principal
- remove_service_principal
- sandbox gov

### CODE REVIEW

File: src/runtime_config/default_actions.js [hunk 1]
```diff
@@ -418,4 +418,29 @@ const actions = new Map([
       }
     ),
   ],
+  [
+    "add_node_code",
```
Comment (jumaffre) [inline src/runtime_config/default_actions.js:422]: Ditto, should we rename the existing hardcoded C++ proposals to match these too?

Comment (achamayou) [reply src/runtime_config/default_actions.js:422]: Let's keep the Lua gov as compatible as possible, as discussed.

File: CMakeLists.txt [hunk 2]
```diff
@@ -500,9 +503,16 @@ if(BUILD_TESTS)
 
   if(ENABLE_JS_GOV)
     set(CONSTITUTION_ARGS
-        --constitution ${CCF_DIR}/src/runtime_config/default_actions.js
-        --constitution ${CCF_DIR}/src/runtime_config/test_actions.js
-        --constitution ${CCF_DIR}/src/runtime_config/constitution.js
+        --constitution
+        ${CCF_DIR}/src/runtime_config/default/actions.js
+        --constitution
+        ${CCF_DIR}/src/runtime_config/test/test_actions.js
```
Comment (achamayou) [inline CMakeLists.txt:509]: I had this as "actions.js" first, but for some bizarre reason, CLI11 seems to deduplicate it with `default/actions`. I will look at this later.

File: metrics.yml [hunk 3]
```diff
@@ -2,7 +2,7 @@ db: "ccf"
 collection: "perf"
 view: "trend"
 columns: 4
-span: 20
+span: 150
```
Comment (achamayou) [inline metrics.yml:5]: Temporary, to display a longer history, to be removed before merging.

File: CMakeLists.txt (hunk 2 above)
Comment (achamayou) [reply CMakeLists.txt:509]: It’s the copy in remote.py.

File: CMakeLists.txt [hunk 4]
```diff
@@ -220,7 +220,10 @@ install(
 
 install(PROGRAMS tests/sandbox/sandbox.sh DESTINATION bin)
 install(FILES tests/sandbox/sandbox_gov.lua DESTINATION bin)
-install(FILES tests/sandbox/sandbox_constitution.js DESTINATION bin)
+install(FILES src/runtime_config/default/actions.js DESTINATION bin)
+install(FILES src/runtime_config/default/validate.js DESTINATION bin)
+install(FILES tests/sandbox/resolve.js DESTINATION bin)
```
Comment (achamayou) [inline CMakeLists.txt:225]:
```suggestion
install(FILES src/runtime_config/sandbox/resolve.js DESTINATION bin)
```

**Review by eddyashton — APPROVED** [review]

**Pull request merged**